File: sentimental-mario-less/mario.py
from cs50 import get_int

while(True):
    height = get_int("Height: ")

    if 0<height<9:
        break

for x in range(height):
    for y in range(height-1, x, -1):
        print(" ", end="")
    for z in range(0, x+1, 1):
        print("#", end="")
    print("")

File: mario-less/mario.c
#include <stdio.h>
#include <cs50.h>


int main()
{
    int height;
    do
        height = get_int("Height: ");
    while (height<=0 || height>=9);

    for (int x = 1; x<=height; x++)
    {
        for(int y=height-1; y>=x; y--)
        {
            printf(" ");
        }

        for (int z = 1; z<=x; z++)
        {
            printf("#");
        }

        printf("\n");
    }
}
